src/libecalc/application/graph_result.py

```diff
@@ -1249,9 +1249,10 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
 
             sub_components.append(obj)
 
-        # When only venting emitters are specified, without a generator set: the installation result is empty.
-        # Ensure that the installation regularity is found, even if only venting emitters are defined:
-        if not installation_results:
+        # When only venting emitters are specified, without a generator set: the installation result
+        # is empty for this installation. Ensure that the installation regularity is found, even if only
+        # venting emitters are defined for one installation:
+        if len(installation_results) < len(asset.installations):
```

Comment:
`<` or `<=` ?

Reply:
should be supplied with a test imo, is that feasible?

Reply:
> `<` or `<=` ?

If I am thinking correctly now: `<` will capture the situations where we have installations with only venting emitters, i.e. installation result is empty for that installation. But, all other cases will have installation results for all installations, or? In any case `<=` will mean that it will always happen, because we will never have more results than installations.

Reply:
I see. I would have considered to get regularities from the "model" for all installations in that case, ie change the way we get regularitieties in general, instead of relying on the inst.result. But I undertand and agree that this also works.